### ISSUE TITLE
MissingDependencyError - This package (or this version) uses `./parser`, 

### ISSUE THREAD

**Issue by irustm** [discoveryjs/jora#9]
https://bundlephobia.com/result?p=jora

MissingDependencyError
This package (or this version) uses `./parser`, but does not specify them either as a dependency or a peer dependency

```
ModuleNotFoundError: Module not found: Error: Can't resolve './parser' in '/tmp/tmp-build/packages/build-jora@1.0.0-alpha.10-6EM/node_modules/jora/src'
```

**Comment by lahmatiy** [comment]
Have no idea why this error happen. `./parser` is a reference to [local module](https://github.com/lahmatiy/jora/blob/master/src/parser.js). Maybe a bug on bundlephobia?

Jora works fine on runkit (https://npm.runkit.com/jora). Here is [example](https://runkit.com/embed/qx3nge0e0y1k):
![image](https://user-images.githubusercontent.com/270491/60394725-8e9f0600-9b31-11e9-9e3c-2e4a1f9055b5.png)

**Comment by lahmatiy** [comment]
Fixed in `1.0.0-alpha.12` by including build prerequisite files into package. It still fails by `jora` search, however works for [`jora@1.0.0-alpha.12`](https://bundlephobia.com/result?p=jora@1.0.0-alpha.12)
